src/apis/MessagesEndpoint.es6.js

```diff
@@ -20,9 +29,10 @@ const parsePostBody = (res, apiResponse, apiRequest) => {
 };
 
 const getPath = (query={}) => {
-  const { subreddit, type } = query;
+  const { subreddit, type, thread } = query;
   const sub = subreddit ? `r/${ subreddit }/` : '';
-  return `${ sub }message/${ type }.json`;
+  const id = thread ? `/${ thread }` : '';
+  return `${ sub }message/${ type }${ id }.json`;
```

Comment:
I think you're probably just being consistent within this file but I don't think we normally put the spaces between the variable and the curlies while doing string interpolation. Can you remove them here and on line 33?